runtime/etc/mli/mli_common_code.c

```diff
@@ -190,6 +191,36 @@ int chpl_mli_pull(void* socket, void* buffer, size_t bytes, int flags) {
   return zmq_recv(socket, buffer, bytes, flags);
 }
 
+// Determine connection information for that socket.
+static
+char * chpl_mli_connection_info(void* socket) {
+  // Determine port used by ZMQ for socket.
+  size_t lenPort = 256;
+  char* portRes = (char *)mli_malloc(lenPort);
```

Comment:
How do we handle (a very unlikely, but still perhaps possible) malloc failure here? I'm still not sure myself.

Reply:
It seems like there's at least a few places in the runtime that choose to ignore that possibility, so I'm going to follow that precedent :)